ui/src/views/Clients/ViewClients/index.jsx

```diff
@@ -121,6 +126,7 @@ export default class ViewClients extends PureComponent {
       description,
       data: { loading, error, clients },
     } = this.props;
+    const { search } = this.state;
```

Comment:
No need to have internal state. We can get this value from the URL. Here's an example where it is done:

https://github.com/taskcluster/taskcluster/blob/d7753390faf752a84b2370f905d07d34257f41d0/ui/src/views/Hooks/ListHooks/index.jsx#L53-L54

Reply:
While you're at it, if you feed the value of the search term to the `defaultValue`, then the search term will show in the search box on page reload.

```jsx
<Search
  // ...
  defaultValue={searchTerm}
/>
```